osmo_jupyter/ysi.py

```diff
@@ -0,0 +1,68 @@
+import pandas as pd
+
+
+def _guard_index_is_datetime(ysi_data):
+    if not isinstance(ysi_data.index, pd.DatetimeIndex):
+        raise ValueError(
+            '''
+            ysi_data must be a DataFrame indexed by pre-parsed timestamps. Get this with, eg.:
```

Comment:
```suggestion
            ysi_data must be a DataFrame indexed by pre-parsed timestamps. Get this with, e.g.:
```
*looks sheepish* :sheep: